nexus-repository-helm/Dockerfile

```diff
@@ -3,15 +3,18 @@ ARG NEXUS_VERSION=3.18.0
 FROM maven:3-jdk-8-alpine AS build
 ARG NEXUS_VERSION=3.18.0
 ARG NEXUS_BUILD=01
+ARG HELM_VERSION=0.0.13
```

Comment:
Not a blocker, but it would be nice if we could find a way to stop the propagation of duplication of "version numbers" in the code base. Not only here, but in pom.xml, etc. Not saying I know how just yet, but ideally there would be one place where the NXRM parent version is specified (e.g. disk file default, overridable via env and/or CLI arg). The same for the Helm plugin version.

Separately, I'm not sure I like the way our current CI setup leaves the most recently released version as the next "development" version. We may decide to fall back to a SNAPSHOT suffix at some point in the future. Yet, another reason to minimize duplication of version values.

Reply:
It's a good concern, but I don't know the fast and correct solution to this problem for now.